docs/how-tos/configure.md

```diff
@@ -117,7 +122,7 @@ Once you have configured your machine, continue to develop an application:
 {{% card link="/how-tos/develop-app/" %}}
```

Comment:
I think we should add fragements here too and then change the next steps intro
```suggestion
{{% card link="/how-tos/develop-app/" %}}
{{% card link="/how-tos/one-to-many/" %}}
```

Reply:
sorry, what do you mean by adding fragments here too? What is the motivation behind directing them to fragments? that would help me understand how to incorporate

Reply:
oh ok i might get it just directing them to one to many how to? lmk what you think

Reply:
Yes that how-to is essentially the guide to fragments.

Please spend a few minutes thinking about the context of what I am saying before asking me to explain it further. You have enough product knowledge to know why fragments should be mentioned to someone who has just configured a machine.

Reply:
i was a bit confused bc that doesn't immediately make sense to me, this is like the very first step of building a machine and then making a fragment to configure a lot of machines is a lot more effort and commitment to using 
Viam. That's why I was asking for more information.

Reply:
your suggestion above makes a lot of sense 👍